schema.sql

```diff
@@ -37,3 +37,33 @@ ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCE
 -- Create an owner foreign key id
 ALTER TABLE animals ADD COLUMN owner_id SMALLINT;
 ALTER TABLE animals ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+-- Create vets table
+CREATE TABLE vets (
+    id serial PRIMARY KEY,
+    name VARCHAR(255),
+    age INT,
+    date_of_graduation DATE
+);
+
+-- Create specializations table (join table for vets and species)
+CREATE TABLE specializations (
+    vet_id INT,
+    species_id INT,
+    PRIMARY KEY (vet_id, species_id),
+    FOREIGN KEY (vet_id) REFERENCES vets(id),
+    FOREIGN KEY (species_id) REFERENCES species(id)
+);
+
+-- Create visits table (join table for animals and vets)
+CREATE TABLE visits (
+    visit_id serial PRIMARY KEY,
```

Comment:
:orange_book: [OPTIONAL] Please Don't add unnecessary context to column names. Column names should be descriptive, but they should also be concise. Avoid adding unnecessary context to column names, such as the table name. For example, instead of `visit_id`, `visit_date`,  you could simply use `id` ,`date`